frappe/website/router.py

```diff
@@ -229,10 +234,26 @@ def get_page_info(path, app, start, basepath=None, app_path=None, fname=None):
 
 	return page_info
 
+def get_frontmatter(string):
+	"""
+	Reference: https://github.com/jonbeebe/frontmatter
+	"""
+
+	fmatter = ""
+	body = ""
+	result = re.compile(r'^\s*(?:---|\+\+\+)(.*?)(?:---|\+\+\+)\s*(.+)$', re.S | re.M).search(string)
+
+	if result:
+		fmatter = result.group(1)
+		body = result.group(2)
+
+	return {
+		"attributes": yaml.load(fmatter),
```

Comment:
```suggestion
		"attributes": yaml.safe_load(fmatter),
```
To avoid following warning.

<img width="935" alt="Screenshot 2020-04-10 at 9 13 40 AM" src="https://user-images.githubusercontent.com/13928957/78960610-36f3e900-7b0c-11ea-8679-bd824b0aacba.png">

Ref: https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation